### DISCUSSION TITLE
How to run MusicBot on each startup (Raspbian)?

### DISCUSSION THREAD

**Discussion by omi-l** [jagrosh/MusicBot#1257] [Questions]
Each time i want to run the bot on my server, i open a bash file with the following command:
`cd /home/orangepi/Botdiscord/`
`./bot.sh`  **it works properlly when i open it manually**
Inside of it is:

`#!/bin/sh
nohup java -jar -Dnogui=true /home/orangepi/Botdiscord/*.jar
`
I tried to open the bash script automatically on crontab but it does not start on the server:
`crontab -e`
`@reboot /home/orangepi/Botdiscord/bot.sh`
What can i do?

**Comment by ipetko96** [comment]
This was already answered here: https://github.com/jagrosh/MusicBot/discussions/704#discussioncomment-943667

From the answer, just follow the  instructions:
I made bash script that will create systemctl service for the Bot.
You have to [configure the Bot](https://github.com/jagrosh/MusicBot/wiki/Setup) before this script.
Just paste it in the console and follow instructions.

#### Copy everything at once, it is one command!
`cd ; curl https://gist.githubusercontent.com/ipetko96/46f14f79391ed2405f8ad3f117d02311/raw/39e7c419a3814cf2813b5c102926060b557eec99/createJMusicBotService.sh --output createJMusicBotService.sh ; sudo bash createJMusicBotService.sh ; rm createJMusicBotService.sh`

It will automatically:
- download [bash script](https://gist.github.com/ipetko96/46f14f79391ed2405f8ad3f117d02311)
- execute as root
- create systemctl service
- set correct permissions
- enable the service
- clean after itself.

Now, after every system reboot, Bot will automatically start.
